lib/coffeescript/kits/paged-publication/initialize.coffee

```diff
@@ -87,6 +96,12 @@ module.exports = (options = {}, callback) ->
             return
 
         callback null, viewer, data
+        if !options.data.details
+            viewer.trigger 'catalogFetched', data.details
```

Comment:
We should never use the word `catalog`. From now on, it's paged publication. The only place where we have catalog is in the actual API URL's.